test/evergreen.yml

```diff
@@ -1128,6 +1142,7 @@ buildvariants:
     test_env_vars: PATH=/opt/mongodbtoolchain/v3/bin:$PATH LD_LIBRARY_PATH=$(pwd)/.libs top_srcdir=$(pwd)/.. top_builddir=$(pwd)
     smp_command: -j $(grep -c ^processor /proc/cpuinfo)
     configure_env_vars: CC=/opt/mongodbtoolchain/v3/bin/gcc CXX=/opt/mongodbtoolchain/v3/bin/g++ PATH=/opt/mongodbtoolchain/v3/bin:$PATH
+    posix_configure_flags: --enable-diagnostic --enable-python --enable-zlib --enable-snappy --enable-strict --enable-static --prefix=$(pwd)/LOCAL_INSTALL
```

Comment:
Should we add `CFLAGS="-ggdb" --enable-silent-rules`, which are currently in the `wiredtiger-pull-request-python` job? And do we need to preload `libeatmydata.so`?

Reply:
These flags are all about debugging information and doesn't change the build itself. I don't want to create a separate compile task just for this test. libeatmydata is currently not available and we have decided to support that when we have it.